src/main/java/com/powsybl/openloadflow/sensi/DcSensitivityAnalysis.java

```diff
@@ -233,13 +233,13 @@ private void createBranchSensitivityValue(LfSensitivityFactor<DcVariableType, Dc
                     sensiValue += contingencyElement.getAlphaForSensitivityValue() * contingencySensitivity;
                 }
                 if (contingencyElement.getElement().getId().equals(functionBranchId)) {
-                    // the monitored branch is in contingency, sensitivity and post-contingency flow equals to zero in any case.
-                    flowValue = 0d;
+                    // the monitored branch is in contingency, sensitivity equals 0 and post-contingency flow equals NaN in any case.
+                    flowValue = Double.NaN;
```

Comment:
Is the post contingency flow NaN 🤔 good 💡,  can we think of something better?
It is clear that after contingency this like is no more active in network
I would actually think of another Enum which would specify this and then even NaN would make sense.
Enum =POST_CONT_OUTOFSERV